src/traccuracy/track_errors/_ctc.py

```diff
@@ -87,9 +88,11 @@ def get_edge_errors(matched_data: Matched):
         logger.warning("Node errors have not been annotated. Running node annotation.")
         get_vertex_errors(matched_data)
 
-    induced_graph = comp_graph.get_subgraph(
-        comp_graph.get_nodes_with_flag(NodeFlag.TRUE_POS)
-    ).graph
+    comp_tp_nodes = comp_graph.get_nodes_with_flag(NodeFlag.TRUE_POS)
+    if len(comp_tp_nodes) == 0:
+        induced_graph = nx.DiGraph()
+    else:
+        induced_graph = comp_graph.get_subgraph(comp_tp_nodes).graph
```

Comment:
Since we're accessing the networkx graph anyway using `.graph`, what do you think about just keeping 

`induced_graph = comp_graph.graph.subgraph(comp_tp_nodes)`

here? This would give us an empty graph for the 0 case without having to do the if statement, and would also avoid the extra overhead of getting the `TrackingGraph` subgraph just to access the nx attribute anyway